content/webapp/components/IIIFViewer/MainViewer.tsx

```diff
@@ -469,39 +464,62 @@ const MainViewer: FunctionComponent = () => {
     }
   }, [canvas]);
 
-  useEffect(() => {
-    // Keeping this very simple for now - I think it'll just apply to audio, video and of course mixed works containing either of them.
-    // I would like to also have it return false for PDFs, but they currently do have a thumbnail image.
-    setHasImagesOnly(!canvases?.find(canvas => !canvas.thumbnailImage?.url));
-  }, []);
+  const currentCanvas = canvases?.[queryParamToArrayIndex(canvas)];
```

Comment:
Noticing this is being done a few times in the file, would it be worth pulling at higher level and not having to redeclare?